Kickstarter-iOS/Features/Comments/Controllers/CommentRepliesViewController.swift

```diff
@@ -198,8 +198,14 @@ final class CommentRepliesViewController: UITableViewController, MessageBannerVi
   }
 
   private func handleCommentCellHeaderTapped(in cell: UITableViewCell, _ author: Comment.Author) {
-    guard AppEnvironment.current.currentUser != nil, featureBlockUsersEnabled() else { return }
-    guard author.isBlocked == false else { return }
+    guard
```

Comment:
This could probably be in a helper func somewhere, it's getting a little long!

Reply:
Yeah, I thought about that but I didn't see a nice place to put it. I'm going to just leave this for now so we can move forward with this, but I'll look into/see if Scott has ideas for where this kind of helper code should live.